proto/frequenz/api/dispatch/v1/dispatch.proto

```diff
@@ -309,6 +309,22 @@ message DispatchFilter {
 
   // Optional filter by update time
   TimeIntervalFilter update_time_interval = 8;
+
+  // Optional filter by partial id matches
+  // If this field is not set, dispatches of any id will be included.
+  // Example:
+  // - To retrieve dispatches with ids containing 123 (e.g.: [5123, 1234, 1235, 123]):
+  //   filter { ids: [123] }
+  // - To retrieve dispatches with ids containing 1 and 5 (e.g.: [5, 1, 15, 51]):
+  //   filter { ids: [1, 5] }
```

Comment:
Shouldn't this say "or" instead of "and"? If it is "and", I would expect 5 and 1 not to match.

Reply:
true, though this will be changed again once UX made up their mind what they even want

Reply:
Then I guess we wait...